README.md

```diff
@@ -10,6 +10,15 @@ distributed networking databus.
 publish and subscribe to the *RTI Connext DDS databus* in Python and other
 languages.
 
+.. note::
```

Comment:
@samuelraeburn, I wasn't sure if this was the right way to update the .md file, which is formatted differently from the otherwise identical .rst file.

Reply:
@alexcamposruiz I think this file is auto-generated from the other one, right?